tests.py

```diff
@@ -334,6 +334,32 @@ def test_unregister_unregisters_multiple(self):
         self.db.unregister_model(self.MyModel)
         assert self.db.model_registry == {}
 
+    def test_add_column_doesnt_break_anything(self):
+        self.db.run("ALTER TABLE foo ADD COLUMN boo text")
+        one = self.db.one("SELECT foo.*::foo FROM foo WHERE bar='baz'")
+        assert one.boo is None
+
+    def test_replace_column_different_type(self):
+        self.db.run("CREATE TABLE grok (bar int)")
+        self.db.run("INSERT INTO grok VALUES (0)")
+        class EmptyModel(Model): pass
+        self.db.register_model(EmptyModel, 'grok')
+        # Add a new column then drop the original one
+        self.db.run("ALTER TABLE grok ADD COLUMN biz text NOT NULL DEFAULT 'x'")
+        self.db.run("ALTER TABLE grok DROP COLUMN bar")
+        # The number of columns hasn't changed but the names and types have
+        one = self.db.one("SELECT grok.*::grok FROM grok LIMIT 1")
+        assert one.biz == 'x'
+        assert not hasattr(one, 'bar')
+
+    @mark.xfail
```

Comment:
I've added the appropriate `raises` parameter. From [pytest's documentation](http://pytest.org/latest/skipping.html#mark-a-test-function-as-expected-to-fail):

> If you want to be more specific as to why the test is failing, you can specify a single exception, or a list of exceptions, in the `raises` argument. Then the test will be reported as a regular failure if it fails with an exception not mentioned in `raises`.